www/app.py

```diff
@@ -11,15 +12,20 @@
 except ImportError:
     print "Adafruit_GPIO lib unavailable"
 
-
+''' Default. Use with rovercode-web running at rovercode.com '''
 ROVERCODE_WEB_REG_URL = "https://rovercode.com/mission-control/rovers/"
+''' Uncomment this line to use with rovercode-web running at beta.rovercode.com '''
```

Comment:
I would suggest using a command line option to choose this. That way the source code doesn't have to be modified to change this location.